src/components/AddBookForm/AddBookForm.js

```diff
@@ -1,18 +1,51 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { bookAddedAction } from '../../redux/books/book';
 
-const AddBookForm = () => (
-  <form className="add-book">
-    <div className="add-book__header">
-      <h3> ADD NEW BOOK</h3>
-    </div>
-    <div className="add-book__controls">
-      <input type="text" placeholder="Book title" />
-      <input type="text" placeholder="Author" />
-    </div>
-    <div className="add-book__actions">
-      <button type="submit">ADD BOOK</button>
-    </div>
-  </form>
-);
+const AddBookForm = () => {
+  const dispatch = useDispatch();
+  const bookObject = {};
+
+  const submitHandler = (event) => {
+    event.preventDefault();
+  };
+
+  const titleInputValueHandeler = (event) => {
+    bookObject.title = event.target.value;
+  };
+
+  const authorInputValueHandler = (event) => {
+    bookObject.author = event.target.value;
+  };
```

Comment:
- [ ] [OPTIONAL] After adding the values to the `bookObject` it would be nice to see an empty form kindly remove/nullify the form after submission.

Reply:
@Wes-Isaac  could you please explain to me more because I did not understand you well?